_inc/client/plans/single-product-backup.jsx

```diff
@@ -35,6 +42,83 @@ export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
 	);
 }
 
+class SingleProductBackupBody extends React.Component {
+	static propTypes = {
+		backupPlanPrices: PropTypes.object,
+		currencySymbol: PropTypes.string,
+		setSelectedBackupType: PropTypes.func,
+		selectedBackupType: PropTypes.string,
+		upgradeLinks: PropTypes.object,
+	};
+
+	constructor() {
+		super();
+
+		this.handleSelectedBackupTypeChange = this.handleSelectedBackupTypeChange.bind( this );
```

Comment:
Do you think it'd be possible to drop binding and switch to using class properties / arrow functions instead?

Reply:
Done in 4ca5cfbb24eb5d39b03847145d9bb532797c3f55